examples/simpleapp/src/main/java/com/horizen/examples/SimpleAppModule.java

```diff
@@ -100,27 +101,28 @@ protected void configure() {
                 .annotatedWith(Names.named("ApplicationState"))
                 .toInstance(defaultApplicationState);
 
+        int storageMaxRollbackDepth = 720 * 2 + 1; //two consensus epochs length + 1
         bind(Storage.class)
                 .annotatedWith(Names.named("SecretStorage"))
-                .toInstance(IODBStorageUtil.getStorage(secretStore));
+                .toInstance(new VersionedLevelDbStorageAdapter(secretStore, storageMaxRollbackDepth));
```

Comment:
At the moment we don't allow sidechain developers to customize consensus epoch length.
Let's make `storageMaxRollbackDepth` as an internal hardcoded parameter of  `VersionedLevelDbStorageAdapter` class.
As for me, developers will be confused about specification of this argument on their side and can do a mistake by passing the value less then `720*2+1`.

Reply:
done